source/agora/consensus/EnrollmentManager.d

```diff
@@ -1300,44 +1248,45 @@ unittest
 
     // create and add the first enrollment
     Enrollment[] enrolls;
-    auto enroll = man.createEnrollment(utxo_set.keys[0]);
+    auto enroll = man.createEnrollment(utxo_set.keys[0], Height(10));
     assert(man.addEnrollment(enroll, key_pair.address, Height(10),
```

Comment:
```suggestion
    assert(man.addEnrollment(enroll, key_pair.address, Height(9),
```

Reply:
Ok I see what you mean now :+1:  This should work I guess?

``` diff
diff --git a/source/agora/consensus/EnrollmentManager.d b/source/agora/consensus/EnrollmentManager.d
index 6ec43d67..e88b56e3 100644
--- a/source/agora/consensus/EnrollmentManager.d
+++ b/source/agora/consensus/EnrollmentManager.d
@@ -1249,17 +1249,17 @@ unittest
     // create and add the first enrollment
     Enrollment[] enrolls;
     auto enroll = man.createEnrollment(utxo_set.keys[0], Height(10));
-    assert(man.addEnrollment(enroll, key_pair.address, Height(10),
+    assert(man.addEnrollment(enroll, key_pair.address, Height(9),
             utxo_set.getUTXOFinder()));
 
     // if the current height is smaller than the available height,
     // we can get no enrollment
-    man.getEnrollments(enrolls, Height(9), &utxo_set.peekUTXO);
+    man.getEnrollments(enrolls, Height(8), &utxo_set.peekUTXO);
     assert(enrolls.length == 0);
 
     // if the current height is greater than or equal to the available height,
     // we can get enrollments
-    man.getEnrollments(enrolls, Height(10), &utxo_set.peekUTXO);
+    man.getEnrollments(enrolls, Height(9), &utxo_set.peekUTXO);
     assert(enrolls.length == 1);
 ```

Reply:
Ok, I appllied the diff above.

Reply:
> Ok I see what you mean now 👍 This should work I guess? Should be identical to what `Ledger` does.

Yes, That's what I meant. Thanks.